Zadanie 12.4 Ożywiamy kanban!/js/Column.js

```diff
@@ -0,0 +1,57 @@
+function Column(id, name) {
+    this.id = id;
+    this.name = name || 'No name given';
+		this.element = createColumn();
+
+		function createColumn() {
+			// TWORZENIE NOWYCH WĘZŁÓW
+			var column = $('<div class="column"></div>');
+			var columnTitle = $('<h2 class="column-title">' + self.name + '</h2>');
+			var columnCardList = $('<ul class="card-list"></ul>');
+			var columnDelete = $('<button class="btn-delete">x</button>');
+			var columnAddCard = $('<button class="column-add-card">Dodaj kartę</button>');
+		}
+		// PODPINANIE ODPOWIEDNICH ZDARZEŃ POD WĘZŁY
+		columnDelete.click(function() {
+			self.deleteColumn();
+		});
+
+		$columnAddCard.click(function(event) {
+			var cardName = prompt("Enter the name of the card");
+			event.preventDefault();
+			$.ajax({
+				url: baseUrl + '/card',
+				method: 'POST',
+				data: {
+				name: cardName,
+				bootcamp_kanban_column_id: self.id
+				},
+				success: function(response) {
+					var card = new Card(response.id, cardName);
+					self.createCard(card);
+    	}
+		});
+	});
+			// KONSTRUOWANIE ELEMENTU KOLUMNY
+		column.append(columnTitle)
+		.append(columnDelete)
+		.append(columnAddCard)
+		.append(columnCardList);
+		return column;
+
+Column.prototype = {
```

Comment:
to powinno wyleciec poza konstruktor

Reply:
niestety dalej to samo :( 
jquery-3.0.0.min.js:4 GET https://kodilla.com/pl/bootcamp-api/board 403 (Forbidden)
send @ jquery-3.0.0.min.js:4
ajax @ jquery-3.0.0.min.js:4
(anonymous) @ App.js:9
jquery-3.0.0.min.js:4 POST https://kodilla.com/pl/bootcamp-api/column 403 (Forbidden)
send @ jquery-3.0.0.min.js:4
ajax @ jquery-3.0.0.min.js:4
(anonymous) @ Board.js:12
dispatch @ jquery-3.0.0.min.js:3
q.handle @ jquery-3.0.0.min.js:3